packages/ember-glimmer/lib/helpers/action.js

```diff
@@ -57,14 +63,15 @@ export class ClosureActionReference extends CachedReference {
       if (!action) {
         throw new EmberError(`An action named '${actionName}' was not found in ${target}`);
       }
+    } else if (action && typeof action[INVOKE] === 'function') {
+      target = action;
+      action = action[INVOKE];
     } else if (actionType !== 'function') {
-      throw new EmberError(`An action could not be made for \`${rawAction}\` in ${target}. Please confirm that you are using either a quoted action name (i.e. \`(action '${rawAction}')\`) or a function available in ${target}.`);
+      // FIXME: Is there a better way of doing this?
```

Comment:
Can you change this to a `TODO:`?  I discussed with @chancancode and this seems okish for now, and we can circle back to it later...